lnwallet/channel.go

```diff
@@ -2558,8 +2560,12 @@ func (lc *LightningChannel) CompleteCooperativeClose(remoteSig []byte) (*wire.Ms
 // DeleteState deletes all state concerning the channel from the underlying
 // database, only leaving a small summary describing metadata of the
 // channel's lifetime.
-func (lc *LightningChannel) DeleteState() error {
-	return lc.channelState.CloseChannel()
+func (lc *LightningChannel) MarkChannelAsFullyClosed() error {
```

Comment:
Hmm, I'd say this method an the one below aren't strictly necessary. Instead the `DeleteState` should untouched, but renamed to something like `MarkAsClosed`.

In line with my prior comments, marking a channel as "fully closed" shouldn't take place _through_ the `OpenChannel` struct, as the channel is no longer "open".